modules/scanning-images.adoc

```diff
@@ -7,26 +7,9 @@
 
 [role="_abstract"]
 
-For version 4.4, {product-title-short} provides two scanners: the StackRox Scanner and Scanner V4. Both scanners can examine images in secured clusters connected in your network. Secured cluster scanning is enabled by default in {osp} environments deployed by using the Operator or when delegated scanning is used. See "Accessing delegated image scanning" for more information.
+{product-title-short} provides two scanners: the StackRox Scanner and Scanner V4. The StackRox Scanner is deprecated. Scanner V4 is enabled by default when installing {product-title-short} on images where Central is installed and on secured clusters.
```

Comment:
Should V4 go first? 
```suggestion
{product-title-short} provides two scanners: the StackRox Scanner and Scanner V4. Scanner V4 is enabled by default when installing {product-title-short} on images where Central is installed and on secured clusters. The StackRox Scanner is deprecated. 
```
Also, I'm not clear on what this means, but I am not target audience, clearly. 

> on images where Central is installed and on secured clusters